docs/apache-airflow/best-practices.rst

```diff
@@ -200,10 +200,12 @@ Some cases of dynamic DAG generation are described in the :doc:`howto/dynamic-da
 Airflow Variables
 -----------------
 
-As mentioned in the previous chapter, :ref:`best_practices/top_level_code`. you should avoid
-using Airflow Variables at top level Python code of DAGs. You can use the Airflow Variables freely inside the
-``execute()`` methods of the operators, but you can also pass the Airflow Variables to the existing operators
-via Jinja template, which will delay reading the value until the task execution.
+Using Airflow Variables yields network calls and database access, so their usage in top-level Python code for DAGs
+should be avoided, as mentioned in the previous chapter, :ref:`best_practices/top_level_code`.
+Their impact on DAG parsing can be mitigated though, by :ref:`enabling caching<config:secrets__use_cache>`
+with a sensible :ref:`ttl<config:secrets__cache_ttl_seconds>`.
```

Comment:
In the docs we at least need to mention this is experimental. I also don't think we should present it as "safe", which is how I read the changes in this doc.